### DISCUSSION TITLE
WithScopes is not a function

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#2800] [Help] [answered]
Hello, i readed docs about withScopes but i can`t use this function, i received error 'withScopes in not a function', is it function available in framework? thanks=) version 5.1.7

**Comment by RomainLanz** [comment, marked as answer]
Hey @sasha172017! 👋 

What version of `@adonisjs/lucid` are you using?

**Reply by sasha172017** [reply]
i use "@adonisjs/lucid": "^14.0.2",

**Reply by thetutlage** [reply]
It's part of the `15.0.1` version https://docs.adonisjs.com/releases/june-2021-release

**Reply by sasha172017** [reply]
Thanks
